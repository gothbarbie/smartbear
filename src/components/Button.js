import styled from 'styled-components'

const Button = styled.button`
  background: ${({ primary }) => (primary ? '#2d9cdb' : '#FFFFFF')};
  border-radius: 2px;
  border: 1px solid ${({ primary }) => (primary ? '#2d9cdb' : '#E0E0E0')};
  color: ${({ primary }) => (primary ? '#F2F2F2' : '#828282')};
  font-size: 18px;
  font-weight: 300;
  padding: 0.5rem 1rem;
  min-width: 120px;
`

export default Button
