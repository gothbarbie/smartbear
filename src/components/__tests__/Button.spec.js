import React from 'react'
import { shallow } from 'enzyme'

import Button from '../Button'

describe('components/Button', () => {
  let component

  beforeEach(() => {
    component = shallow(<Button />)
  })

  it('renders Button', () => {
    expect(component).toMatchSnapshot()
  })
})
