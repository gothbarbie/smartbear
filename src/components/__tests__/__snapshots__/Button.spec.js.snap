// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Button renders Button 1`] = `
<button
  className="sc-bdVaJa fKRDEG"
/>
`;
